conjure-java-core/src/main/java/com/palantir/conjure/java/ConjureAnnotations.java

```diff
@@ -50,6 +52,10 @@ public static AnnotationSpec getConjureGeneratedAnnotation(Class<?> clazz) {
         return deprecation.isPresent() ? DEPRECATED : ImmutableList.of();
     }
 
+    public static ImmutableList<AnnotationSpec> override(boolean override) {
```

Comment:
Curious why this is returning a list instead of just the `AnnotationSpec`. We could still use the `MethodSpec.Builder#addAnnotation` builder method which does not require a collection.

Reply:
This way we can use the method fluently. If override is false, we'd have to break out of the fluent `.builder().setX(X).build()` flow.

Reply:
Didn't think of that but makes sense!